java-manta-client-unshaded/src/main/java/com/joyent/manta/config/BaseChainedConfigContext.java

```diff
@@ -882,6 +907,7 @@ public boolean equals(final Object other) {
                 && Objects.equals(verifyUploads, that.verifyUploads)
                 && Objects.equals(uploadBufferSize, that.uploadBufferSize)
                 && Objects.equals(skipDirectoryDepth, that.skipDirectoryDepth)
+                && Objects.equals(downloadContinuations, that.downloadContinuations)
```

Comment:
It seems a bit odd that you added `downloadContinuations` to the equals() method, but not to the hashCode() method.

Reply:
Silly mistake, sorry.